projects/ci-cd.md

```diff
@@ -0,0 +1,130 @@
+# CI/CD Observability Semantic Conventions Working Group
+
+## Description
+
+The goal of this working group is define the semantic conventions for CI/CD Observability and outline 
+a path forward for the OpenTelemetry community. 
+
+### Why
+
+The goal of this project is to establish standard semantic conventions for Continuous
+Integration (CI) and Continuous (CD) observability. This will provide a common language
+and standardized formats for CI/CD observability, enabling the community to observe CI/CD systems.
+
+This will broaden the target audience of OpenTelemetry to Release Engineering and DevOps teams,
+further cementing OpenTelemetry as the industry standard Observability framework.
+
+The timing is ripe to start now. The [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files)
+has been open since January of 2023 and with the recent changes to the OTEP process, the KubeCon talk, and vendor acknowledgements, there's momentum available to carry this forward. The industry 
+is heavily looking for solutions and watching the related OTEP with interest.
+
+## Deliverables
+
+The CI/CD observability working group is planning to deliver the [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files) with prototypes in the following areas:
+
+Only a few technologies are named here, so this is not all encompassing. As we build out these conventions, specifications, and prototypes we'll want to ensure that Observability is done within:
+
+* The CI system (SCMs like GitHub/GitLab)
+* The tools used within the CI system (Maven/Go)
+* The CD system (SCMs if handling deploys / Keptn / ArgoCD)
+
+Current, non-complete list of Deliverables:
+
+* Semantic Convention Updates
+* Specification Updates (as applicable)
+* [GitHub](https://github.com)
+* [GitLab](https://gitlab.com)
+* [Jenkins](https://www.jenkins.io/)
+* [Keptn](https://lifecycle.keptn.sh/)
+
+> Working prototypes are partially available in these systems, but have been done outside of the context of this semantic conventions. Thus these prototypes would be updated based on semantic conventions defined.
+
+### Related OTEP(s)
+
+A related OTEP that is of great interest to the CI/CD OTEP is that of [Environment Variables as Context Propagators](https://github.com/open-telemetry/opentelemetry-specification/issues/740).
+This OTEP might be driven through other working groups, and isn't the direct focus of this working group, but is has wide arching impacts to enabling distributed tracing in batch systems. 
+As we work to define semantic conventions for CI/CD observability, this OTEP will without a doubt come into play in some form and thus is being listed as related but not a direct deliverable.
+
+
+### Existing Related Solutions
+
+The following is a list of prototypes, plugins, components, and CI/CD related tooling that attempt to address CI/CD observability in some form. 
+
+> This is not a complete list but can be used for reference.
```

Comment:
Tekton supports distributed tracing https://github.com/tektoncd/community/blob/main/teps/0124-distributed-tracing-for-tasks-and-pipelines.md